### PR TITLE
Fix duplicates in discovery returns due to dumb bug in OR conditions.

### PR DESCRIPTION
**Pull request by alvacouch**
I made a dumb syntax mistake in the fix that repeated results. FIxed. Sorry. 

### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. Each result in discover appears once.

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|
| NEW SECURITY HOTSPOTS | 0 |
| NEW BUGS | 0 |
| NEW CODE SMELLS | 0 |
| NEW VULNERABILITIES | 0 |
| NEW DUPLICATED LINES DENSITY | 0.0 |

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4208](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4208)

File: hs_discover/views.py [hunk 1]
```diff
@@ -70,47 +70,47 @@ def get(self, request, *args, **kwargs):
             qs = request.query_params
             filters = json.loads(qs.get('filter'))
             # filter values expect lists, for example discoverapi/?filter={"owner":["Firstname Lastname"]}
-            if filters.get('author'):
+            if filters.get('author') and len(filters['author']) > 0:
```
Comment (sblack-usu) [inline hs_discover/views.py:73]: you could simplify to `len(filters.get('author', '')) > 0`

**Review by sblack-usu — APPROVED** [review]
I left one comment that would clean It up a tiny bit but I'm not going to hold this up if you don't want to make the change.

File: hs_discover/views.py (hunk 1 above)
Comment (alvacouch) [reply hs_discover/views.py:73]: filters.get('author') can return None, crashing this.

Comment (alvacouch) [reply hs_discover/views.py:73]: I'm going nuts today. I really can't handle this change right now.

**Pull request merged**